OPHD/States/MapViewStateTurn.cpp

```diff
@@ -509,11 +505,13 @@ void MapViewState::nextTurn()
 
 	mPreviousMorale = mCurrentMorale;
 
+	produceFood();
```

Comment:
this was done slightly earlier before, in `StructureManager::update` -> `StructureManager::updateStructures` -> `Agridome::think`. Food levels are not used between here and the prior production point.